gtest/Cargo.toml

```diff
@@ -17,3 +19,4 @@ colored = "2.0.0"
 env_logger = "0.9.0"
 path-clean = "0.1.0"
 wasm-instrument = { version = "0.1" }
+wasmtime = { version = "0.35.1", default-features = false}
```

Comment:
we should use similar package as inside wasmtime backend (I'm about features here, versions are the the same).